index.html

```diff
@@ -328,18 +327,18 @@ <h1 class="form__h1 poppins">
                 <ul class="form__list">
                     <li class="form__list1">
                         <input class="poppins form__list__style" name="name" type="text" id="Firstname"
-                            placeholder="Najash" maxlength="30" required />
+                            placeholder="Najash" maxlength="30" aria-label="User Name" required/>
                     </li>
                     <li>
                         <input class="poppins form__list__style" type="email" name="email" id="email"
-                            placeholder="email@something.com" required />
+                            placeholder="email@something.com" aria-label="Email" required />
                     </li>
                     <li class="form__list2">
                         <textarea class="poppins form__list__style" name="textarea__form" id="textarea__form" cols="30"
-                            rows="10" maxlength="500" placeholder="Write your message here" required></textarea>
+                            rows="10" maxlength="500" aria-label="Enter Message" placeholder="Write your message here" required></textarea>
                     </li>
                     <li class="form__button__li">
-                        <button type="submit" class="form__button poppins">Get In Touch</button>
+                        <button type="submit" class="form__button poppins" aria-label="Get In Touch">Get In Touch</button>
                     </li>
                 </ul>
             </form>
```

Comment:
![link-to-the-change (copy)](https://user-images.githubusercontent.com/93039359/187475249-c8ef7040-b590-46bb-938b-4c3f7ef190c0.png)

- [ ] Please add links in your pull request summary to those lines of code which you have changed in your html :+1:
Hint: You can see this pull [request summary](https://github.com/orozCoding/myportfolio/pull/8) to get a good idea :+1: